static/css/slick.css

```diff
@@ -4,7 +4,7 @@
 
 .slick-list { position: relative; overflow: hidden; display: block; margin: 0; padding: 0; }
 .slick-list:focus { outline: none; }
-.slick-loading .slick-list { background: #fff url("./ajax-loader.gif") center center no-repeat; }
+.slick-loading .slick-list { background: #fff url("/static/img/ajax-loader.gif") center center no-repeat; }
```

Comment:
We're using the [cf-icon-update](https://github.com/cfpb/regulations-site/pull/711) with a CSS animation in other parts of cf.gov. Would it make sense to use that here instead of the gif?